### PR TITLE
Explain use of `pg_config`

### PR DESCRIPTION
**Pull request by BanzaiMan**
Resolves https://github.com/travis-ci/travis-ci/issues/9011.

### CODE REVIEW

File: user/database-setup.md [hunk 1]
```diff
@@ -199,6 +199,23 @@ before_install:
...
+apt package `postgresql-server-dev-X.Y` for the version of PostgreSQL you are using in order
+to obtain meaningful output from the command.
+
+For example:
+
+```yaml
+addons:
+  apt:
+    packages:
+      - postgresql-server-dev-9.4
```
Comment (plaindocs) [inline user/database-setup.md]: Just to clarify, the X.Y / 9.4 needs to match the existing postgre they are using? So either the default, or the one they specify intentionally?

Comment (BanzaiMan) [reply user/database-setup.md]: Any one that the user intends to use. This assumes:

```yaml
addons:
  postgresql: "9.4"
```

Perhaps I can add that.

Comment (BanzaiMan) [reply user/database-setup.md]: That is to say:

```yaml
addons:
  postgresql: "9.4"
  apt:
    packages:
      - postgresql-server-dev-9.4
```

Comment (plaindocs) [reply user/database-setup.md]: How about 

> If your builds rely on the `pg_config` command, you need to install an additional
apt package `postgresql-server-dev-X.Y` where X.Y matches the version of PostgreSQL you are using.

Comment (BanzaiMan) [reply user/database-setup.md]: That sounds great! :+1:

**Pull request merged**